Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -311,7 +311,9 @@ public virtual void Animate(string sequenceName, float duration = -1f) {
             return;
         }
 
-        idleTask = MovementState.TryEmote(sequence.Name, false, duration);
+        bool isIdle = sequenceName.Contains("idle", StringComparison.OrdinalIgnoreCase);
```

Comment:
I'm a little on the fence about blanket calling animations with `idle` in the name an idle task, behavior-wise, but it is worth a shot. If it breaks anything we can take a closer look when we discover it.